spec/bolt/node/winrm_spec.rb

```diff
@@ -279,6 +279,20 @@ def stub_winrm_to_raise(klass, message)
     end
   end
 
+  it "passes arguments to the task in the powershell way", winrm: true do
+    contents = <<-PS
+      param ($Name, $Age)
+
+      Write-Host "$Name is $($Age+5) years old."
+    PS
+    arguments = { :Name => 'John Doe',
```

Comment:
I'd suggest passing the args out of order from their definition to make sure you're not accidentally getting parameters matching by position.

Reply:
Did that even before reading this ... ;-)